xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlaySSA.h

```diff
@@ -34,7 +34,6 @@ class CDVDOverlaySSA : public CDVDOverlay
   {
     replace = true;
     m_libass = libass;
-    libass->Acquire();
```

Comment:
Agree that Acquire does look wrong and removing it also fixes the libass not being freed.